application/F3DOptionsParser.cxx

```diff
@@ -364,7 +364,9 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp7, "interaction-test-play", "", "Path to an interaction log file to play interaction events from when loading a file", appOptions.InteractionTestPlayFile, LocalHasDefaultNo, MayHaveConfig::YES,"<file_path>");
     // clang-format on
 
+    cxxOptions.positional_help("<input>");
```

Comment:
```suggestion
    cxxOptions.positional_help("file1 file2 ...");
```
To keep the old `Usage: f3d [OPTION...] file1 file2 ...` instead of making it `Usage: f3d [OPTION...] <input>` ?